clusterloader2/pkg/config/cluster.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+// ClusterConfig is a structure that represents cluster description.
+type ClusterConfig struct {
+	KubeConfigPath string `json: kubeConfigPath`
+	NodeCount      int    `json: nodeCount`
```

Comment:
Why do you need NodeCount?
@shyamjvs @krzysied

Reply:
Discussed earlier in https://github.com/kubernetes/perf-tests/pull/192#pullrequestreview-147560699. We should eventually get rid of it.

Reply:
TBH my personal take on it (if we indeed need this variable) is to get this from 'list nodes' instead of this.

Reply:
Yes - that's what I was going to tell.
[BTW - the link above doesn't work for me]